README.md

```diff
@@ -66,6 +66,14 @@ But for some identifiers might have more. Check [their implementation](https://g
 
 For `URN`s, please check the [URN gem documentation](https://github.com/altmetric/urn) to see all the available options.
 
+### DOIs
+
+DOI's are notoriously tricky to pin down in a regex, since the actual regex could feasibly be `10\.{\d}+/\S+` in order to match all possible DOIs.
```

Comment:
Also, DOI prefixes _aren't_ necessarily numeric only, as the registrant code supports subdivisions, e.g., `10.123.56` - see https://www.doi.org/doi_handbook/2_Numbering.html#2.2.2